skyportal/utils/offset.py

```diff
@@ -156,6 +157,101 @@ def get_ztfref_url(ra, dec, imsize, *args, **kwargs):
 }
 
 
+def calculate_best_position(
+    photometry, fallback=(None, None), how="snr", max_offset=0.5, sigma_clip=4.0
+):
+    """Calculates the best position for a source from its photometric
+       points
+
+    Parameters
+    ----------
+    photometry : list
+        List of Photometry objects
+    fallback : tuple, optional
+        The position to use if something goes wrong here
+    how : str
+        how to weight positional data:
+          snr = use the signal to noise
+          err = use the photometric uncertainty
+    max_offset : float, optional
+        How many arcseconds away should we ignore discrepant points?
+    sigma_clip : float, optional
+        Remove positions that are this number of std away from the median
+    """
+
+    if not isinstance(photometry, list):
+        log(
```

Comment:
Why not raise this?

Reply:
This is not an error - it's certainly possible to have an Obj without any associated photometry (e.g. DESI). It's graceful to just return the fallback position (ie. the Obj ra/dec)